src/autogluon/cloud/predictor/timeseries_cloud_predictor.py

```diff
@@ -175,13 +178,18 @@ def predict_real_time(
         Pandas.DataFrame
         Predict results in DataFrame
         """
+        self.id_column = id_column or self.id_column
+        self.timestamp_column = timestamp_column or self.timestamp_column
+        self.target_column = target or self.target_column
```

Comment:
What will happen if both `self.target_column is None` and `target` is None?

Reply:
I think in the `TabularPredictor` API does not require `target` at the moment, that's why I made it optional. See this tutorial, e.g.
```
predictor = TimeSeriesPredictor(prediction_length=14).fit(train_data)
```.
It has been handled https://github.com/autogluon/autogluon/blob/bda6174f4a1fb8398aef4f375d9eacfd29bb46d9/timeseries/src/autogluon/timeseries/predictor.py#L179